runtime/local/src/chain_extensions.rs

```diff
@@ -34,3 +36,8 @@ impl RegisteredChainExtension<Runtime> for DappsStakingExtension<Runtime> {
 impl RegisteredChainExtension<Runtime> for XvmExtension<Runtime> {
     const ID: u16 = 01;
 }
+
+/// Based on chain-extension registry https://github.com/paritytech/chainextension-registry
+impl<W: WeightInfo> RegisteredChainExtension<Runtime> for AssetsExtension<Runtime, W> {
+    const ID: u16 = 18678;
+}
```

Comment:
We are not using that implementation though.

So what's the correct way to handle this?

Reply:
Are you sure this is the right way to go?

If your implementation is the _correct_ one, should it be _official_ instead of the SuperColony one?